python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -485,12 +502,14 @@ class RandomForestRegressor(Base):
         concat_model_handle = concatenate_trees(deref(model_handles))
 
         concat_model_ptr = <size_t> concat_model_handle
-        return ctypes.c_void_p(concat_model_ptr).value
-
-    def _concatenate_model_bytes(self, concat_model_handle):
-        cdef uintptr_t model_ptr = <uintptr_t> concat_model_handle
-        concat_model_bytes = save_model(<ModelHandle> model_ptr)
-        self.model_pbuf_bytes = concat_model_bytes
+        self.treelite_handle = ctypes.c_void_p(concat_model_ptr).value
+        cdef uintptr_t model_ptr = <uintptr_t> self.treelite_handle
+        cdef vector[unsigned char] pbuf_mod_info = \
+            save_model(<ModelHandle> model_ptr)
```

Comment:
As `concat_model_ptr` already holds a `uintptr_t` to the pointer we want, we can just use that directly without converting back from Python.

```suggestion
        cdef vector[unsigned char] pbuf_mod_info = \
            save_model(<ModelHandle> concat_model_ptr)
```